docs/source/content/unsupervised/_img/clustering.rst

```diff
@@ -0,0 +1,222 @@
+##########
```

Comment:
you accidentally copied this to the img folder it seems

Reply:
Yep. Thanks for catching that. I just removed the extra copy.